test/k8s-integration/main.go

```diff
@@ -665,7 +665,7 @@ func runTestsWithConfig(testParams *testParameters, testConfigArg, reportPrefix
 	kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--skip-regex=%s", testParams.testSkip))
 	// kubetest uses 25 as default value for ginkgo parallelism (--nodes).
 	kubeTest2Args = append(kubeTest2Args, "--parallel=25")
-	kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--test-args=%s %s", testConfigArg, reportArg))
+	kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--test-args=%s %s", testConfigArg, windowsArgs))
```

Comment:
only if testParams.platform is windows, we should add windowsArgs?

Reply:
That's right - corrected.